src/eigen.jl

```diff
@@ -1,242 +1,107 @@
-# MIT License: Copyright (c) 2016: Andy Ferris.
-# See LICENSE.md for further licensing test
+# Specify conversion to static arrays for 2nd order tensors
+@inline StaticArrays.SMatrix(a::Tensor{2,dim}) where{dim} = SMatrix{dim,dim}(a)
+@inline StaticArrays.SMatrix(a::SymmetricTensor{2,dim}) where{dim} = Symmetric(SMatrix{dim,dim}(a))
```

Comment:
Could be internal functions instead
```suggestion
to_smatrix(a::Tensor{2, dim, T}) where {dim, T} = SMatrix{dim, dim, T}(a)
to_smatrix(a::SymmetricTensor{2, dim, T}) where {dim, T} = Symmetric(SMatrix{dim, dim, T}(a))
```